pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -156,19 +138,50 @@ func doUpdate(ctx *zedmanagerContext,
 
 		// Trigger the rollback process
 		if status.SnapStatus.HasRollbackRequest {
-			err := triggerRollback(ctx, *status)
-			status.SnapStatus.HasRollbackRequest = false
+			snappedAppInstanceConfig, err := triggerRollback(ctx, status)
```

Comment:
This is returning the config from the snapshot, right?
Doesn't that mean that we should replace all of of the config?
At some point in time will the controller send us the old config used at the time of the snapshot? Or is there logic in zedmanager to ignore any changes to AppInstanceConfig while we are running a rollbacked config?
(We still have the fact that the A